forge-gui/res/cardsfolder/b/banes_invoker.txt

```diff
@@ -0,0 +1,6 @@
+Name:Bane's Invoker
+ManaCost:1 W
+Types:Creature Human Cleric
+PT:2/2
+A:AB$ Pump | Cost$ 8 | TargetMin$ 0 | TargetMax$ 2 | NumAtt$ +2 | NumDef$ +2 |KW$ Flying | ValidTgts$ Creature | TgtPrompt$ Select target Creature | SpellDescription$ Up to two target creatures each get +2/+2 and gain flying until end of turn. 
```

Comment:
Add `PrecostDesc$ Wind Walk —`
Insert space before `KW$`
`TgtPrompt$ Select up to two target creatures`